src/js/components/PageSection/index.d.ts

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { BoxProps } from '../Box/index';
+import { BackgroundType } from '../../utils';
+
+export interface PageSectionProps {
+  fullBackground?: BackgroundType;
```

Comment:
I have `fullBackground` as a separate prop because I wanted to allow setting the background on the PageContent through background in addition to setting a background that isn't confined by the page kind width. For example in the below picture the image is being set with the `fullBackground` prop and the pink background is being set with the `background` prop. So if we do `background={{ fill: "horizontal" }}` this would only allow setting either the full background or the constrained background not both.

![Screen Shot 2022-02-18 at 1 52 05 PM](https://user-images.githubusercontent.com/54560994/154759621-0564bf2e-4e13-4b36-9df3-5c3c5b580f69.png)

Reply:
Hmm... do we have an actual design example where we would want both? I'm thinking that would be very rare, and that the caller could always fill the PageSection with a `<Box fill background={} />` in that case.